src/main/java/usecases/submit_word/SwInputData.java

```diff
@@ -0,0 +1,49 @@
+package usecases.submit_word;
+
+/**
+ * The data that the SwInteractor needs in order to determine if a word can be added,
+ * and then if it is valid, then add the word.
+ */
+public class SwInputData {
+    private String word;
+
+    private String playerId;
+
+    /**
+     * The constructor of SwInputData.
+     * @param word the word that has been submitted.
+     * @param playerId the Id of the player that attempted to submit the word.
```

Comment:
minor (unimportant) comment: for more readability, consider using tab to evenly space parameter name and description